app/src/main/res/values/dimens.xml

```diff
@@ -1,10 +1,25 @@
 <resources>
-    <!-- Default screen margins, per the Android Design guidelines. -->
-    <dimen name="activity_horizontal_margin">16dp</dimen>
-    <dimen name="activity_vertical_margin">16dp</dimen>
-    <dimen name="fab_margin">16dp</dimen>
-    <dimen name="app_bar_height">200dp</dimen>
-    <dimen name="item_width">200dp</dimen>
-    <dimen name="text_margin">16dp</dimen>
-    <dimen name="spacing_medium">16dp</dimen>
+
+    <dimen name="margin_extra_small">4dp</dimen>
```

Comment:
i think you misunderstood @iamareebjamal .
what he meant is use general names for same padding or layout margin like small medium etc. i find that more appropriate.